packages/styles/src/components/_banner.scss

```diff
@@ -0,0 +1,89 @@
+.lc-banner {
+  box-sizing: border-box;
+  overflow: hidden;
+  padding: 20px;
+  position: relative;
+  width: 100%;
+  word-break: break-word;
+
+  &--small {
+    max-width: 400px;
+
+    .lc-banner__content {
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+      width: 100%;
+
+      &-text {
+        margin: 14px 0 0;
+        max-width: 246px;
+      }
+    }
+  }
+
+  &--medium {
+    max-width: 800px;
+
+    .lc-banner__content {
+      display: flex;
+      justify-content: flex-start;
+      width: 100%;
+
+      &-text {
+        margin: 0 30px 0 16px;
+        max-width: 640px;
+      }
+    }
+  }
+
+  &--large {
+    .lc-banner__content {
+      display: flex;
+      justify-content: flex-start;
+      width: 100%;
```

Comment:
I would extract it one level higher...and add `column` type one for `--small` variant. It will reduce the amount of the css code.

Reply:
Let's handle this during the refactoring process and focus on migration only.